deploy-board/deploy_board/webapp/helpers/baseimages_helper.py

```diff
@@ -149,6 +149,10 @@ def get_image_update_events_by_cluster(request, cluster_name):
         event['status'] = generate_image_update_event_status(event)
     return events
 
+def get_latest_succeeded_image_update_event_by_cluster(request, cluster_name):
```

Comment:
Does this return `Optional[int]` or `Optional[float]`? `finish_time` type is unclear